lib_expense/src/main/java/com/example/expenselogger/lib_expense/domain/contract/ExpenseContract.kt

```diff
@@ -0,0 +1,10 @@
+package com.example.expenselogger.lib_expense.domain.contract
+
+import com.example.expenselogger.lib_expense.domain.model.Expense
+
+internal interface ExpenseContract {
+    fun saveExpenseEntry(expense: Expense)
```

Comment:
Is there a reason you suffixed all the functions with `Entry`? If not, then I think it can be removed since the function names will still be descriptive without it.

Reply:
I'll remove it from the functions